app/(context)/index.jsx

```diff
@@ -0,0 +1,21 @@
+'use client'
+import React, { useState, createContext } from 'react';
+//This basically initializes an instance of a context API
+export const Context = createContext();
```

Comment:
```suggestion
export const GlobalContext = createContext();
```
I think Context is a bit too generic